dask_glm/logistic.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+from scipy.optimize import fmin_l_bfgs_b
+import dask.array as da
+
+
+def logistic_regression(X, y, alpha, rho, over_relaxation):
+    N = 5
+    (m, n) = X.shape
+
+    z = np.zeros([n, N])
+    a = np.zeros([n, N])
+    beta_old = da.from_array(np.zeros([n, N]), chunks=(2, 1))
+    beta = np.zeros((2, 5))
+    u = da.from_array(a, chunks=(2, 1))
+    MAX_ITER = 100
+    ABSTOL = 1e-4
+    RELTOL = 1e-2
+
+    for k in range(MAX_ITER):
+        beta_x = y.map_blocks(local_update, X, beta_old.T, z[:, 0], u.T, rho, chunks=(5, 2)).compute()
```

Comment:
If we are only using dask.array for `map_blocks` then it might also make sense to use dask.delayed.  Keeping all arrays as numpy arrays (or lists of numpy arrays for `X` and `y`) might make things more explicit.